website/docs/patterns/description-list-patterns/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,113 @@
+
+When using the callout value style, use a grid with 24px vertical gaps and 48px horizontal gaps. Set each list item’s width to fill for even visual spacing.
+
+![Two rows with three columns. The gaps vertically show 24px and horizontally shows 48px.](/assets/patterns/description-list-patterns/description-list-vertical-callout-grid-gaps.png)
+
+#### Wrapping
+
+If content is too condensed, wrap content to a new row.
+
+!!! Do
+
+Wrap content to ensure legibility of each list item.
+
+![Two rows with five columns. The first row has each column filled, while the second row has one. Information fits comfortably in each column](/assets/patterns/description-list-patterns/description-list-vertical-do-grid.png)
+
+!!!
+
+!!! Dont
+
+Force list items into a single row, instead consider arranging items into uniform rows. This practice promotes visual consistency and enhances readability.
+
+![One row with seven columns. The information in each column wraps inconsistently and has varied heights](/assets/patterns/description-list-patterns/description-list-vertical-dont-grid.png)
+
+!!!
+
+### Horizontal
+
+Horizontal orientation should only be used with the standard styling of headings and values. Using two sizes in the horizontal orientation throws off the visual balance of the content in a description list.
+
+#### Spacing
+
+Use 16px gaps between the heading and value and each row pairing. Components can be used in place of the value, but only one component should be used per description list item.
+
+![Five rows of information showing a vertical gap between the heading and value of 16px and between each row as 16px.](/assets/patterns/description-list-patterns/description-list-horizontal-standard-grid-gaps.png)
+
+In rare instances where the value slot requires additional rows for context, use a 4px gap. The content within the value slot can have multiple rows. Consider the visual hierarchy of the overall grouping during implementation.
+
+![The gap between the heading and value is 16px. The value has two rows and a 4px gap between each row.](/assets/patterns/description-list-patterns/description-list-horizontal-value-multiple-rows.png)
+
+##### Exceptions
+There may be occasions when the standard gap between items in a description list needs to be adjusted. For example, a description list used as a legend requires a 4px row gap instead of the standard 16px, as the values are closely related.
+
+![Highlighting list items in a horizontal orientation stacked vertically on top of each other with a 4px gap between each other.](/assets/patterns/description-list-patterns/description-list-logical-gap.png)
+
+In horizontal orientations, use your best judgment to adjust the gaps between list items for optimal UI representation.
+
+#### Column width
```

Comment:
Here's where I think things a bit confusing (we chatted about this briefly today), but the `Description List` is the pattern consisting of a list of `Description List / Items` (thinking about organization here), so is this actually the **Item** that is organized horizontally, and the **List** that is organized vertically? Might be good to be a bit more clear about that (may impact some other parts of the documentation).